pyregion/ds9_region_parser.py

```diff
@@ -152,119 +153,54 @@ def convert_attr(self, l):
                 yield l1, c1
 
     @staticmethod
-    def sky_to_image(l, header, rot_wrt_axis=1):
-        """
+    def sky_to_image(shapelist, header):
```

Comment:
I'll add to the Changelog. I made this change because the 1-3 letter variables make it harder to understand what the code is doing. The `l` parameter, the class for this function, and the module itself isn't documented, so I don't think there was any intention to make this a public API. Nice to see that `regions` uses `__all__` pretty judiciously to make this stuff clear.